pages/dkp/kommander/2.2/install/air-gapped/index.md

```diff
@@ -74,7 +74,25 @@ It is possible to push new charts as well:
 Or push a new bundle:
 
    ```bash
-   kommander push chart-bundle [chartsTarball]
+   export KUBECONFIG=${MANAGEMENT_KUBECONFIG}
```

Comment:
```suggestion
      kommander push chart-bundle [chartsTarball]
      export KUBECONFIG=${MANAGEMENT_KUBECONFIG}
```

Reply:
This was an oops addition, correcting this deleted line item & please ignore this .md file, otherwise.